src/components/cartItem/cartItem.styles.css

```diff
@@ -0,0 +1,50 @@
+.cart-item-wrapper {
+  background-color: #ffffff;
+  border-radius: 10px;
+  padding: 12px;
+  min-height: 30px;
+  margin-bottom: 20px;
+}
+
+.cart-item-wrapper .cart-item-information {
+  border-right: 2px solid #cdcdcd;
+  position: relative;
+  width: 93%;
+  display: inline-block;
+  margin-right: 20px;
+  vertical-align: middle;
+}
+
+.cart-item-wrapper .cart-image-item {
+  display: inline-block;
+  width: 60px;
+  height: 60px;
+  margin-right: 12px;
+  vertical-align: top;
+}
+
+.cart-item-wrapper .cart-item-info {
+  display: inline-block;
+  vertical-align: middle;
+  text-transform: capitalize;
+  width: 70%;
+  margin-right: 12px;
```

Comment:
I'm seeing a lot of inline-blocks. Why aren't you using flexbox?

Reply:
Flexbox is not supported in IE all browsers I can implement same with flex box as well .. Earlier I did it with flexbox but during optimization I thought this can come as a review comment that's why I avoided it